tests/integration/test_github.py

```diff
@@ -111,6 +114,14 @@ def test_get_releases(self):
 
         assert len(releases) >= 9
 
+    def test_create_release(self):
+        release = self.hello_world_project.create_release(
+            tag="0.3.0", name="test", message="testing release"
+        )
+        assert release.tag_name == "0.3.0"
+        assert release.title == "test"
+        assert release.body == "testing release"
```

Comment:
Maybe, we can test also the increase in the number of releases. (`before + 1 == after`)